data/hails.txt

```diff
@@ -700,9 +747,12 @@ phrase "friendly civilian"
 		"rotten fruit"
 		"sludge"
 		"unprocessed sewage"
+		"nuclear waste"
+		#add more smelly stuff
 	word
 		". My"
 		", and now my"
+		", and since my"
```

Comment:
Doesn't make sense: "Dropped off my cargo of limburger cheese, and since my beautiful ship's cargo hold totally reeks!"

Reply:
it was supposed to be since then my